funnel/assets/js/utils/helper.js

```diff
@@ -402,6 +402,25 @@ const Utils = {
     }
     return false;
   },
+  getFaviconHTML(icon, iconSize = 'body', baseline = true, cssClassArray=[]) {
```

Comment:
Favicon != FA Icon. Function needs to be renamed. Also, we should upgrade from FA5 to FA6.